### ISSUE TITLE
Please add https://tamilrockers.ws to the list of trackers

### ISSUE THREAD

**Issue by jkailasam** [Jackett/Jackett#1831]
Is it possible to add "http://tamilrockers.nz/index.php?" to the list of trackers? This tracker is for mainly Indian movies. 

thanks

**Comment by kaso17** [comment]
If you know a little HTML/CSS you can try to add it yourself.
See https://github.com/Jackett/Jackett/wiki/Definition-format for details.
If you've any questions feel free to ask.

If the tracker already provides some kind of API (e.g. gazelle based trackers) please let us know. Then the Cardigann definition approach might be not not the best solution

**Comment by garfield69** [comment]
site is not suitable for cardigann processing, requires native c# indexer.

**Comment by garfield69** [comment]
domain gone.

**Comment by anburocky3** [comment]
Domain renamed: https://tamilrockers.ws/

**Comment by nero12** [comment]
hi, whats the status on this ? was it possible to add it to jackett ?

**Comment by garfield69** [comment]
waiting on a c# volunteer to write an indexer for this site.

**Comment by ngosang** [comment]
Duplicate of #5570
